components/shared/Editor.tsx

```diff
@@ -526,10 +524,12 @@ export default function MockupEditor() {
           x: x - imagePosition.x,
           y: y - imagePosition.y,
         };
+        e.preventDefault();
       } else if (text && isPointInText(x, y)) {
         setIsDragging(true);
         setDragTarget("text");
         offsetRef.current = { x: x - textPosition.x, y: y - textPosition.y };
+        e.preventDefault();
```

Comment:
What if we remove this e.preventDefault(), does the text stays inside the border but things are selectable again?